cloudSetup/sshTunnelSql.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+ssh "ec2-user@$EXASOL_MANAGEMENT_IP" -L 8563:n11:8563
```

Comment:
These are nice :+1: 

Do Java tests start these files?

Reply:
no. Java does it using a Java ssh lib. These are just for debugging